deploy/example.yaml

```diff
@@ -47,6 +47,14 @@ rules:
   - create
   - update
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - configmap
+  resourceNames:
+  - aws-iam-authenticator
```

Comment:
It looks like the ConfigMap mapper is hard coded to watch the [`aws-auth`](https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/pkg/mapper/configmap/configmap.go#L59), can you correct that?

Reply:
Ok, will get the example to have that changed too.